app/src/main/java/com/github/studydistractor/sdp/procrastinationActivity/AddProcrastinationActivityActivity.kt

```diff
@@ -83,12 +103,29 @@ class AddProcrastinationActivityActivity : Hilt_AddProcrastinationActivityActivi
      */
     private fun createNewActivity(name: String, description: String) {
         if (TextUtils.isEmpty(name) || TextUtils.isEmpty(description)) {
-            Toast.makeText(this@AddProcrastinationActivityActivity, "Please fill the blanks", Toast.LENGTH_SHORT).show()
+            Toast.makeText(
+                this@AddProcrastinationActivityActivity,
+                "Please fill in the blanks",
+                Toast.LENGTH_SHORT
+            ).show()
             return
         }
 
-//        TODO: add the possibility to add a location to the activity
         val activity = ProcrastinationActivity(name, description, null, null)
-        service.postProcastinationActivities(activity)
+        service.postProcastinationActivities(activity,
+            {
+                Toast.makeText(
```

Comment:
Maybe do a function to notify the user to avoid duplication of code (suggestion)